extension/app/src/components/conversation/ConversationContainer.tsx

```diff
@@ -256,31 +256,54 @@ export function ConversationContainer({
     setGreeting(getRandomGreetingForName(user.firstName));
   }, [user]);
 
+  if (activeConversationId) {
```

Comment:
I know this duplicates some code but honestly I think it's way easier to read and change 🙈